Source/WebKit/Shared/API/APIError.h

```diff
@@ -51,6 +51,7 @@ class Error : public ObjectImpl<Object::Type::Error> {
         Cancelled = 302,
         FileDoesNotExist = 303,
         HTTPSUpgradeRedirectLoop = 304,
+        HTTPSOnlyHTTPURL = 305,
```

Comment:
HTTPNavigationWithHTTPSOnly?

How about something like this for the string:
"Navigation failed because it was to an HTTP URL with HTTPS-Only enabled"